src/features/transfer-ownership/TransferOwnership.constants.ts

```diff
@@ -0,0 +1,11 @@
+export enum Step {
+	DETAILS,
+	CONFIRM,
+	TRANSACTION_APPROVAL,
+	TRANSACTION_IN_PROGRESS,
+	COMPLETE,
+}
+
+export enum StepText {
+	TRANSACTION_ALERT = 'This transaction is about to be seared upon the blockchain. There’s no going back.',
```

Comment:
I think this could probably just go in the component it's rendered in, since it isn't shared between multiple components.

Reason being - if you were trying to debug something that was to do with the Transactions page, and you wanted to search for the `This transaction is about...` string, it would take you here rather than the component it's in. Probably not the end of the world, but small DX things are nice.

Reply:
Ohh I c, my bad, if it's used in two steps makes sense to have it here